resources/puppetlabs/lein-ezbake/template/foss/ext/default.erb

```diff
@@ -11,12 +11,11 @@ JAVA_ARGS="<%= EZBake::Config[:java_args] %>"
 # These normally shouldn't need to be edited if using OS packages
 USER="<%= EZBake::Config[:user] %>"
 GROUP="<%= EZBake::Config[:group] %>"
-INSTALL_DIR="/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>"
```

Comment:
grepping for INSTALL_DIR it looks like this variable is used in other places and only set here, so may want to leave this defined and just not use interpolation in the BOOTSTRAP_CONFIG below?